lib/jsmidgen.js

```diff
@@ -393,8 +393,15 @@ var Midi = {};
 		var trackCount = this.tracks.length.toString(16);
 
 		// prepare the file header
-		var bytes = File.HDR_CHUNKID + File.HDR_CHUNK_SIZE + File.HDR_TYPE0;
+		var bytes = File.HDR_CHUNKID + File.HDR_CHUNK_SIZE;
 
+		// set Midi type based on number of tracks
+		if (parseInt(trackCount) > 1) {
```

Comment:
Good comments.  I added the radix as you suggested.  I think that line 393 needs to be in hex because of how trackCount is used on line 406.

The link you provided suggests that type 2 isn't popular, so maybe we should not worry about supporting it at this point.  Also, I get what you mean about having a default of type 1, but it really isn't too hard to have the type determined by the number of tracks.  It would be interesting to see what dingram thinks.  Thanks for your comments!